src/parser/tools/parser_test.cpp

```diff
@@ -92,8 +92,8 @@ int main(int argc, char** argv)
         return 1;
     }
 
-    std::string path = *prefix + "/" + *treebank + "/treebank-3/parsed/mrg/"
-                       + *corpus;
+    std::string path
+        = *prefix + "/" + *treebank + "/treebank-3/parsed/mrg/" + *corpus;
```

Comment:
Will building paths like this be compatible on Windows? (and all the other places that we do this)

Maybe we need a `filesystem::join_path`.